docs/tutorials/tokens/erc20.md

```diff
@@ -18,25 +18,25 @@ nav_order: 1
 ---
 
 ## Previous steps: Start your environment
-If you haven't started a FireFly stack already, please go back to the previous step and read the guide on how to [Start your environment](./setup_env.md).
+If you haven't started a FireFly stack already, please go to the Getting Started guide on how to [Start your environment](./setup_env.md). This will set up a token connector that works with both ERC-20 and ERC-721 by default.
 
 [← ② Start your environment](setup_env.md){: .btn .btn-purple .mb-5}
 
-The default Token Connector that the FireFly CLI sets up is for ERC-20 and ERC-721. If you would lik eto use one of these contracts, all you need to do is go through the first two steps in the Getting Started guide.
+## Use the built in sample token factory
+
+If you are using the default ERC-20 / ERC-721 token connector, when the FireFly CLI set up your FireFly stack, it also deployed a token factory contract. When you create a token pool through FireFly's token APIs, the token factory contract will automatically deploy an ERC-20 or ERC-721 contract, based on the pool `type` in the API request.
+
+<div style="color: #ffffff; background: #ff7700; padding: 1em; border-radius: 5px;">⚠️ <span style="font-weight: bold;">WARNING</span>: The default token contract that was deployed by the FireFly CLI is only provided for the purpose of learning about FireFly. It is <span style="font-weight: bold;">not</span> a production grade contract. If you intend to deploy a production application using tokens FireFly you should research token contract best practices. For details, <a style="color: #ffffff;" href="https://github.com/hyperledger/firefly-tokens-erc20-erc721/blob/main/solidity/contracts/TokenFactory.sol">please see the source code</a> for the contract that was deployed.</div>
```

Comment:
I think the link here needs to be adjusted since things moved into a "sample" folder.